app/components/govuk_component/table_component/cell_component.rb

```diff
@@ -75,7 +75,7 @@ def default_classes
     class_names(
       "#{brand}-table__#{class_suffix}",
       "#{brand}-table__#{class_suffix}--numeric" => numeric?,
-      width => width?,
+      self.class.widths[width] => width?,
```

Comment:
If the widths are moved to a constant it might be neater to do this in the initializer with

```
@width = WIDTHS.fetch(width, nil)
```